LoRaEngine/modules/LoRaWanPktFwdModule/start_pktfwd.sh

```diff
@@ -20,7 +20,7 @@ fi
 #get current architecture for the mess processor
 arch="$(uname -m)"
 if [[ $arch != *"arm"* ]]; then
-    if [ -z "$SPI_DEV" ]; then
+    if [ -z "$SPI_DEV" ] || [ "$SPI_DEV" == '$PKT_FWD_SPI_DEV' ]; then
```

Comment:
nit: we could add some debug output to indicate the value was not set.

Reply:
Good idea! I will add logs indicate we are taking the default